create_ancient.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
```

Comment:
ok, then I will drop it for now. I recently added the scripts "lands", which can be used instead of python lands/generator.py. You need to run python setup.py install or python setup.py develop to use it